lib/grinder_tools.dart

```diff
@@ -175,6 +176,58 @@ class Tests {
   }
 }
 
+/// A wrapper around the `test` package. This class is used to run your unit
+/// tests.
+class Test {
+  final PubApp _test = new PubApp.local('test');
+
+  Test();
+
+  /// Run the tests in the current package. See
+  /// https://pub.dartlang.org/packages/test.
+  ///
+  /// [name] is substring of the name of the test to run. Regular expression
+  /// syntax is supported. [plainName] is a plain-text substring of the name of
+  /// the test to run. [platform] is the platform(s) on which to run the tests.
+  /// Available values are `vm` (default), `dartium`, `content-shell`, `chrome`,
+  /// `phantomjs`, `firefox`, `safari`. [concurrency] controls the number of
+  /// concurrent test suites run (defaults to 4). [pubServe] is the port of a
+  /// pub serve instance serving `test/`.
+  void test({String name, String plainName, List<String> platforms,
```

Comment:
I think we should support `platforms` being a `String` either instead of or in addition to `List<String>` to get the full power of the [platform selector syntax](https://github.com/dart-lang/test#platform-selector-syntax).  I think once https://github.com/dart-lang/test/issues/23 is fixed, it will probably expose an enum instead, so that'd be one reason to maybe leave out the `List<String>` option for now until that shakes out.